### PR TITLE
AssImp port update for glTF2 sparse accessors support.

### PR DESCRIPTION
**Pull request by zyndor**
Signed-off-by: György Straub <g,straub@partner.samsung.com>

### CODE REVIEW

File: ports/assimp/portfile.cmake [hunk 1]
```diff
@@ -1,14 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO assimp/assimp
-    REF v5.0.0
-    SHA512 0f73b6e961cd8455d6b6c8c10ed8b99485d846c96377b5d4fcc3b83f737647207c1306aa3dd51dad9654fbfa61bfe1119b34646f90288ae7ecab45efa6fa418a
```
Comment (pwisbey) [inline ports/assimp/portfile.cmake:5]: Is there any reason why we don't have a dalihub/assimp repo?  Permissions issue?

Comment (zyndor) [reply ports/assimp/portfile.cmake:5]: I'm not sure if there's a permissions issue, it just didn't occur to me to try to create under dalihub as the change wasn't strictly related to / required for DALi itself.

Comment (zyndor) [reply ports/assimp/portfile.cmake:5]: (Sorry, my comment is about assimp and I might have misunderstood your question: as for vcpkg, I don't know if I can create a fork (of a fork) under the same organization that already has one, so I just used the account that is mine.)

Comment (pwisbey) [reply ports/assimp/portfile.cmake:5]: I would prefer to do everything in dalihub, instead of people's personal accounts

Comment (pwisbey) [reply ports/assimp/portfile.cmake:5]: Sorry, I missed Victor's version of this fix (http://slp-info.sec.samsung.net/gerrit/#/c/3437504/). I'll try to test / push that instead...

**Comment by vcebollada** [comment]
Resolved by adding a patch to the assimp port

**Pull request closed without merging**